CONTRIBUTING.md

```diff
@@ -69,11 +69,11 @@ Downloaded OpenAPI models are placed in `resources/models/raw/<category>/<api-na
 
 A wide variety of refactoring processes are run on the raw schemas to get them ready to be used in the following step: code generation. The refactoring process code is in `SchemaVersion::refactor()`.
 
-Certain schema changes need to be made manually, because they are too specific or too cumbersome to make in the code. These modifications are defined in the `resources/modifications.json` file, which is explained more thoroughly [below](#modificationsjson).
+Certain schema changes need to be made manually, because they are too specific or too cumbersome to make in the code. These modifications are defined in the [modifications.json](https://github.com/highsidelabs/saloon-sdk-generator/tree/master/src/resources/metadata/modifications.json) file, which is explained more thoroughly [below](#modificationsjson).
```

Comment:
I think this should be `resources/metadata/modifications.json`

Reply:
Fixed, it is indeed clearer

Reply:
Thanks, sorry...copied most of this over from https://github.com/shipstream/fedex-rest-php-sdk since I designed the two libraries very similarly, but missed a couple things